index.html

```diff
@@ -25,6 +25,8 @@
     <link type="text/css" rel="stylesheet" href="res/nouislider.min.css">
     <link type="text/css" rel="stylesheet" href="res/styles.css">
 
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.15.1/moment.min.js"></script>
```

Comment:
I don't have strong views. Really this should be in the node app but we need something working asap and refactor later